drivers/fuel_gauge/CMakeLists.txt

```diff
@@ -2,6 +2,6 @@
 
 add_subdirectory_ifdef(CONFIG_SBS_GAUGE_NEW_API		sbs_gauge)
 add_subdirectory_ifdef(CONFIG_MAX17048		max17048)
+add_subdirectory_ifdef(CONFIG_ACT81461_ALPC		act81461)
```

Comment:
You will need to rebase with the 3.4.0 changes and note here that there is now only only 1 space after the CONFIG name.

Reply:
I was thinking of another file, so this is fine